cli/src/main.rs

```diff
@@ -234,8 +261,33 @@ fn compile_once(world: &mut SystemWorld, command: &CompileSettings) -> StrResult
     match typst::compile(world) {
         // Export the PDF.
         Ok(document) => {
-            let buffer = typst::export::pdf(&document);
-            fs::write(&command.output, buffer).map_err(|_| "failed to write PDF file")?;
+            match command.image_settings {
+                Some(ImageSettings { pixel_per_pt }) => {
+                    let fill = Color::WHITE;
+                    let file_stem = command.output.file_stem().unwrap().to_str().unwrap();
+                    let extension = command.output.extension().unwrap();
+                    // render for each frame and save it with the file stem `{file_stem}-{i}` where i is the page number
+                    for (i, frame) in document.pages.iter().enumerate() {
+                        typst::export::render(frame, pixel_per_pt, fill)
+                            .save_png(
```

Comment:
this function generates pretty big PNGs I _think._ if that is true, we could maybe use the `png` crate or something like that to generate better ones.